File: 백준/Bronze/1264. 모음의 개수/모음의 개수.java
import java.util.*;
public class Main{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        while(true){
            String str = sc.nextLine().toLowerCase();
            if(str.equals("#"))
                break;
            int cnt = 0;
            for(int i = 0; i < str.length(); i++){
                if(str.charAt(i) == 'a' || str.charAt(i) == 'e' || str.charAt(i) == 'i' || 
                str.charAt(i) == 'o' || str.charAt(i) == 'u')
                    cnt++;
            }
            System.out.println(cnt);
        }
        sc.close();
    }
}

File: 백준/Bronze/1264. 모음의 개수/모음의 개수.py
while True:
    s = input().lower()
    if s == '#':
        break

    cnt = 0
    for i in s:
        if i == 'a' or i == 'e' or i == 'i' or i == 'o' or i == 'u':
            cnt += 1
    print(cnt)